recipes/geemap/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "geemap" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 311b4c935575b41819c30d30ac01b7788e1f1d61e996734fb5f4a44945c6b32a
+
+build:
+  number: 0
+  entry_points:
+    - geemap=geemap.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - bqplot
+    - click
+    - earthengine-api
+    - folium >=0.10.1
+    - ipyleaflet >=0.12.2
+    - ipynb-py-convert
+    - pip
+    - python
+  run:
+    - bqplot
+    - click
+    - earthengine-api
+    - folium >=0.10.1
+    - ipyleaflet >=0.12.2
+    - ipynb-py-convert
+    - python
+
+test:
+  imports:
+    - geemap
+  commands:
+    - geemap --help
+
+about:
+  home: "https://github.com/giswqs/geemap"
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
Please add license file. If it is not part of the tarball, you can copy it from the repo.